arangod/Pregel/Algos/AIR/AIR.cpp

```diff
@@ -0,0 +1,143 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2020 ArangoDB GmbH, Cologne, Germany
```

Comment:
Format of copyright headers was changed in devel a while ago for all files, and should be done for all new files introduced here too, e.g.
```suggestion
/// Copyright 2014-2020 ArangoDB GmbH, Cologne, Germany
/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
```
It is fine if we do this later using a script. It does not need to be done in this PR. Just mentioning it here so we don't forget.